__tests__/app.test.js

```diff
@@ -65,6 +65,7 @@ describe('GET /api/articles/:article_id', () => {
             expect(receivedTimestamp).toBe(1604394720000);
             expect(body.article.votes).toBe(0)
             expect(body.article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
+            expect(body.article.comment_count).toBe("2")
```

Comment:
It's generally bad practise to alter previous tests when we add new functionality - rather we should add a separate test to check this new property has be added to the object 🕵️